core/src/Kokkos_View.hpp

```diff
@@ -116,6 +116,15 @@ struct BasicViewFromTraits {
   using type =
       BV::BasicView<element_type, extents_type, layout_type, accessor_type>;
 };
+
+// Helper function to deal with cases where the data handle is
+// not convertible to element_type* such as in Sacado
+template <class PointerType>
+KOKKOS_INLINE_FUNCTION constexpr auto ptr_from_data_handle(
+    const PointerType& handle) {
```

Comment:
why not

```suggestion
    PointerType *handle) {
```

Rather than the static assert?

Reply:
Yeah I guess we could do that (with renamed PointerType), not sure what would be better error message in the long run? We could add to the static_assert something like "If you customize View including a custom data_handle_type, you must provide an ADL findable overload for this function".

Reply:
maybe call it `HandleType`?